packages/wix-ui-core/src/components/radio-button/RadioButton.tsx

```diff
@@ -163,13 +167,15 @@ export class RadioButton extends React.Component<
     this.props.onHover({ value: this.props.value, ...event });
   };
 
-  onFocus = () => {
+  onFocus = (event: React.FocusEvent<HTMLInputElement>) => {
     this.setState({ focused: true, focusVisible: !this.focusedByMouse });
+    this.props.onFocus && this.props.onFocus({ ...event });
```

Comment:
Could be changed to
```suggestion
    this.props.onFocus && this.props.onFocus(event);
```
but not critical 😊